src/NServiceBus.Hosting.Tests/APIApprovals.Approve.approved.txt

```diff
@@ -5,7 +5,9 @@
 [assembly: System.Runtime.Versioning.TargetFrameworkAttribute(".NETFramework,Version=v4.5.2", FrameworkDisplayName=".NET Framework 4.5.2")]
 namespace NServiceBus
 {
+    [System.ObsoleteAttribute(@"Roles are obsoleted and should not be used. The functionality of AsA_Server, and AsA_Publisher has been made defaults in the core, see: https://docs.particular.net/nservicebus/hosting/nservicebus-host/?version=host_6#roles-built-in-configurations. Will be treated as an error from version 7.1.0. Will be removed in version 8.0.0.", false)]
     public interface AsA_Client { }
+    [System.ObsoleteAttribute(@"Roles are obsoleted and should not be used. The functionality of AsA_Server, and AsA_Publisher has been made defaults in the core, see: https://docs.particular.net/nservicebus/hosting/nservicebus-host/?version=host_6#roles-built-in-configurations. Will be treated as an error from version 7.1.0. Will be removed in version 8.0.0.", false)]
```

Comment:
no guarantee this doco url will exist for the same period that this error might be seen by users